### PR TITLE
firejail: update to 0.9.58.

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

File: srcpkgs/firejail/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 build_style=gnu-configure
 configure_args="--enable-apparmor"
+conf_files="/etc/firejail/* /etc/apparmor.d/local/firejail-local"
 makedepends="libapparmor-devel"
 short_desc="SUID security sandbox program"
 maintainer="Duncaen <duncaen@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://firejail.wordpress.com"
-#changelog="https://raw.githubusercontent.com/netblue30/firejail/master/RELNOTES"
+changelog="https://raw.githubusercontent.com/netblue30/firejail/${version}/RELNOTES"
```
Comment (Duncaen) [inline srcpkgs/firejail/template]: `changelog` is still not a real field and I prefer to keep it at master.

File: srcpkgs/firejail/template [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # Template file for 'firejail'
 pkgname=firejail
-version=0.9.56
-revision=2
+version=0.9.58
+revision=1
 build_style=gnu-configure
 configure_args="--enable-apparmor"
+conf_files="/etc/firejail/* /etc/apparmor.d/local/firejail-local"
```
Comment (Duncaen) [inline srcpkgs/firejail/template]: Its really not necessary to move this around. https://github.com/chneukirchen/xtools/pull/115

File: srcpkgs/firejail/template (hunk 1 above)
Comment (prspkt) [reply srcpkgs/firejail/template]: Sure, restored.

File: srcpkgs/firejail/template (hunk 2 above)
Comment (prspkt) [reply srcpkgs/firejail/template]: Ah, just trying to silence xlint.

**Comment by prspkt** [comment]
@Duncaen All set!

**Comment by Duncaen** [comment]
`conf_files` is now duplicated.

**Pull request merged**